HedgeModManager/Languages/zh-CN.xaml

```diff
@@ -229,12 +229,12 @@
     <system:String x:Key="GameSonicLostWorld">索尼克 失落世界</system:String>
     <system:String x:Key="GameSonicForces">索尼克 力量</system:String>
     <system:String x:Key="GamePuyoPuyoTetris2">魔法气泡 特趣思 俄罗斯方块 2</system:String>
-    <system:String x:Key="GameTokyo2020">2020东京奥运</system:String>
+    <system:String x:Key="GameTokyo2020">索尼克在 2022 东京奥运会</system:String>
```

Comment:
Revert this change, it's for the game on Steam.
https://steamdb.info/app/981890/

Reply:
Sorry, my bad